pkg/validate/lambda_validator.go

```diff
@@ -0,0 +1,91 @@
+package validate
+
+import (
+	"fmt"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/interfaces"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"os"
+	"strconv"
+)
+
+const (
+	lambdaAccountType        		= "lambda"
```

Comment:
Nitpicking, here is a mix of tabs and spaces, I think it should be either spaces or tab.

Reply:
fixed